desmod/config.py

```diff
@@ -261,6 +261,10 @@ def factorial_config(base_config, factors, special_key=None):
         When specified, a key/value will be inserted into the generated
         configuration dicts that identifies the "special" (unique) key/value
         combinations of the specified `factors` used in the config dict.
+    :param int only_factor:
+        When specified, only a single config will be yielded; the config will
+        contain only the single factor at the specified index of the list of
+        the cartesian product of the unrolled factors.
```

Comment:
I remain against plumbing this `only_factor` kwarg into `factorial_config()`. We can isolate this capability to `simulate_factors()`. I know that imposes a somewhat larger runtime cost, but I see that cost as marginal and the conceptual complexity of spreading this feature across two modules as the more important concern.